### ISSUE TITLE
COCO evaluation results start outputting zero while training

### ISSUE THREAD

**Issue by WalterYeYint** [tensorflow/models#9004]
I have been training ssd_mobilenet_v2_coco_2018_03_29 using custom dataset and default pretrained weights provided. Issue is while training COCO evaluation always giving mAP=0 while the loss is getting decreased in each epoch.

Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000

I have been training fine using the same .ipynb script until last week when the training process starts outputting mAP and AR of zero values (my guess is due to the new version of python-pil).
I also noticed the error message "Colab is experiencing issues connecting to Drive, and we are actively investigating" last week just before the error occurs, but I don't think it's the cause of the issue.

Any help is appreciated.

**Comment by wuchichung** [comment]
same problem here. #8982

**Comment by WalterYeYint** [comment]
[SOLVED]: My script has been pulling tensorflow models repository everytime it runs. Downloaded that repository from the last time my script was working solved the problem.
